Moose Development/Moose/Wrapper/Controllable.lua

```diff
@@ -3659,3 +3665,51 @@ function CONTROLLABLE:OptionRestrictBurner(RestrictBurner)
   end
 
 end
+
+--- Sets Controllable Option for A2A attack range for AIR FIGHTER units.
+-- @param #CONTROLLABLE self
+-- @param #number range Defines the range 
+-- @usage Range can be one of MAX_RANGE = 0, NEZ_RANGE = 1, HALF_WAY_RMAX_NEZ = 2, TARGET_THREAT_EST = 3, RANDOM_RANGE = 4. Defaults to 3. See: https://wiki.hoggitworld.com/view/DCS_option_missileAttack
+function CONTROLLABLE:OptionAAAttackRange(range)
+  self:F2( { self.ControllableName } ) 
+  -- defaults to 3
+  local range = range or 3
+  if range < 0  or range > 4 then
+    range = 3
+  end       
+  local DCSControllable = self:GetDCSObject()
+  if DCSControllable then
+    local Controller = self:_GetController()
+    if Controller then 
```

Comment:
Usually, I'd say you cannot have enough nil checks. However, in this case that check is already done in the :SetOption() function. So it could be removed here.

Reply:
Noted for the next time!